Begin
`a` has been called!
1
catch(@e) where {: see(I, $x) :}
ERROR {: #206fe716-0c6d-4b31-b13b-2bd5ec8b8aa9 &gt;: { see(i,#a) } :}
on Fired
ensure
#a
End of `Enter`
